generate_types.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# The @types directory can be regenerated with this script
+
+# Install all necessary .gir files
+echo "Installing gir files..."
+sudo apt install libatk1.0-dev:amd64 \
```

Comment:
move the installs to a `setup.sh` or similar.

Any particular reason the amd64 variants are required?